smart-contracts/contracts/DigitalaxAuction.sol

```diff
@@ -72,12 +106,27 @@ contract DigitalaxAuction is Context, ReentrancyGuard {
     /// @notice globally and across all auctions, the amount by which a bid has to increase
     uint256 public minBidIncrement = 0.1 ether;
 
-    constructor(DigitalaxAccessControls _accessControls, DigitalaxGarmentNFT _garmentNft) public {
+    /// @notice global bid withdrawal lock time
+    uint256 public bidWithdrawalLockTime = 20 minutes;
+
+    /// @notice global platform fee, assumed to always be to 1 decimal place i.e. 120 = 12.0%
+    uint256 public platformFee = 120;
+
+    /// @notice where to send platform fee funds to
+    address payable public platformFeeRecipient;
+
+    constructor(
+        DigitalaxAccessControls _accessControls,
+        DigitalaxGarmentNFT _garmentNft,
+        address payable _platformFeeRecipient
+    ) public {
         accessControls = _accessControls;
         garmentNft = _garmentNft;
+        platformFeeRecipient = _platformFeeRecipient;
     }
 
     // TODO add test for creating an action, cancelling it, creating it again - confirm flow works as expected
+    // TODO add test for increasing bid -> placeBid() -> placeBid() again to bump it
```

Comment:
still needed?

Reply:
Ye reckon so, these two cases still need to be confirmed